### ISSUE TITLE
Render incline=x% and incline=-x% not just up/down

### ISSUE THREAD

**Issue by DaveE128** [mattesCZ/mtbmap#106]
It would be great to render positive incline=x% in the same way as currently rendered with incline=up and negative as down. This doesn't seem to be working at present as far as I can see.

Thanks.

**Comment by mattesCZ** [comment]
Yes, we render only incline=up/down. Thanks for pointing this out. We should take percentage values higher than some threshold into account. Maybe higher than 8%?

**Comment by DaveE128** [comment]
Yes, that sounds like an appropriate threshold to me. Thanks.
